### DISCUSSION TITLE
Question about MAISI training batch size

### DISCUSSION THREAD

**Discussion by Mors20** [Project-MONAI/MONAI#8096] [Q&A]
Hi,

Thank you for your excellent work on MAISI. I’ve noticed that the model can generate images with flexible dimensions, and I’m curious about the training process that allows for this flexibility. From my understanding, training with images of varying sizes would typically require a batch size of 1, which can be challenging, as batch size is crucial for diffusion model training. It seems the paper doesn't go into detail on this aspect, so I would appreciate any insights you can provide.

Best regards,
Haoshen

**Comment by KumoLiu** [comment]
Hi @dongyang0122 @guopengf and @Can-Zhao, could you please share some comments on this one? Thanks in advance.

**Comment by guopengf** [comment]
Hi @Mors20, we provide a detailed training description in supplementary section B Additional Implementation Details (https://arxiv.org/pdf/2409.11169v1). Specifically, the batch size is set to 1 per GPU.
